pkg/antctl/raw/traceflow/command.go

```diff
@@ -127,19 +144,19 @@ func runE(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("error when creating Traceflow, is Traceflow feature gate enabled? %w", err)
 	}
 	defer func() {
-		if option.waiting {
+		if !option.nowait {
```

Comment:
Could you explain about this change?

Reply:
I hoped to keep the boolean options consistent and the default value to false. But if there are other cases to use waiting, I can revert the change.

Reply:
OK. Just want to know the purpose of this change.
@lzhecheng Any comments here?

Reply:
No strong opinion here.